components/arm/client.go

```diff
@@ -135,23 +141,12 @@ func (c *client) ModelFrame() referenceframe.Model {
 }
 
 func (c *client) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
-	if c.model == nil {
-		return nil, errArmClientModelNotValid
-	}
-	resp, err := c.JointPositions(ctx, nil)
-	if err != nil {
-		return nil, err
-	}
-	return c.model.InputFromProtobuf(resp), nil
+	return c.JointPositions(ctx, nil)
 }
 
 func (c *client) GoToInputs(ctx context.Context, inputSteps ...[]referenceframe.Input) error {
-	if c.model == nil {
-		return errArmClientModelNotValid
-	}
 	for _, goal := range inputSteps {
-		err := c.MoveToJointPositions(ctx, c.model.ProtobufFromInput(goal), nil)
-		if err != nil {
+		if err := c.MoveToJointPositions(ctx, goal, nil); err != nil {
```

Comment:
Can we add a TODO for these to update to MoveThroughJointPositions when available?